Editor/SnapZone/SnapZoneSettings.cs

```diff
@@ -35,20 +35,35 @@ public class SnapZoneSettings : ScriptableObject
         /// </summary>
         [Tooltip("This color is used when a valid object is hovering the snap zone.")]
         public Color ValidationColor = new Color32(0, 255, 0, 50);
+        
+        /// <summary>
+        /// This color is used when an <see cref="InteractableObject"/> is hovering a <see cref="SnapZone"/>.
+        /// </summary>
+        [Tooltip("This color is used when an invalid object is hovering the snap zone.")]
```

Comment:
The comment and the tooltip contradict themself.

Reply:
fixed